chia/simulator/block_tools.py

```diff
@@ -146,7 +147,7 @@
         "BLOCKS_CACHE_SIZE": 340 + 3 * 50,  # Coordinate with the above values
         "SUB_SLOT_TIME_TARGET": 600,  # The target number of seconds per slot, mainnet 600
         "SUB_SLOT_ITERS_STARTING": 2**10,  # Must be a multiple of 64
-        "NUMBER_ZERO_BITS_PLOT_FILTER": 1,  # H(plot signature of the challenge) must start with these many zeroes
+        "NUMBER_ZERO_BITS_PLOT_FILTER": 2,  # H(plot signature of the challenge) must start with these many zeroes
```

Comment:
without this change, too many plots pass the plot filter and CHIP-13 is likely to reject all test plots, preventing us from generating long test chains